File: pset1/mario/more/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int n = get_int("Height: ");
    while(n > 23 || n < 0)
    {
        n = get_int("Height: ");
    }
    for( int i = 0; i < n; i++){
        int x = 0;
        int y = 0;
        int z = 0;
        while(z < n - i - 1){
            printf(" ");
            z++;
        }
        while(x < i + 1){
            printf("#"); 
            x++;
            
        }
        printf("  ");
        while(y < i + 1){
            printf("#");
            y++;
            
        }
        printf("\n");
    }
}

File: pset6/mario/mario.py
import cs50
#ask use for integer for pyramid height
height = cs50.get_int("Enter Pyramid Height: ")
while height > 23 or height < 0:
    height = cs50.get_int("Enter Pyramid Height: ")

def pyramid(rows):
    for row in range(rows):
        print(" "*(rows - (row + 1)), end="")
        print("#"*(row + 1), end="")
        print("  ", end="")
        print("#"*(row + 1))

pyramid(height)